src/styles/views/Game.scss

```diff
@@ -9,51 +9,57 @@
 
 .game {
   &.container {
```

Comment:
If you do the following:

```js
 .game {
  &.container {
    margin: 1em auto; //Add this, will make sure that the whole thing is in the center
    border-radius: $borderRadius;
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: $background;
    box-shadow: $dropShadow;
  }
}
```
The outer background board (grey), not the word board (black) will be put in the center.